_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Team epages 6 Offsite"
+date: "2016-07-04 13:37:42"
+image: blog-header/ep6offsite2016_back.jpg
+categories: events
+authors: ["Sandra", "Oliver"]
+---
+
+From June 21st to 22nd the team epages 6 went on an offsite -- in Jena! How is it an offsite if you stay in the city you work in? Well, due to the size of the team and various team members having family obligations it was difficult finding a date and a location for this offsite.
```

Comment:
...it was difficult finding a date and a location for this offsite. > it was difficult to find a date and location for this offsite.